_docs/modules.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Separating your code into Modules
+category: Imports and Modules
+---
+
+Modules in Glu allow you to organize and reuse code across different files and projects. Glu provides flexible import mechanisms that allow you to control how external files and their content are imported. This flexibility is essential for managing dependencies, keeping your code clean, and avoiding name conflicts.
+
+In this article, we'll explore different ways to import modules in Glu and discuss why it's important to use them effectively.
+
+## Basic Module Import
+
+The simplest way to import a module in Glu is using the `import` statement followed by the module's filename.
+
+For example, let's say you have a file called `file`:
+
+```glu
+func function() {
+    std::print("Hello from file!");
+}
+```
+
+To use this function in another file, simply import the file and call the function with the module's name as a prefix:
+
+```glu
+import file;
+
+func main() {
+    file::function(); // This will print "Hello from file!"
+}
+```
+Here, `file::function()` is used to call `function` from the `file` module. The `::` operator is used to access functions, constants, and other definitions from the imported module.
```

Comment:
This section lacks explanations regarding how a module is created.

You should at least explain that in Glu, each file represents a module. A module is implicitly created for each file, will include everything contained within the file, and will be named after it.

Without this simple explanation, the examples are not entirely clear.